src/flibustaApiHelper.ts

```diff
@@ -1,16 +1,18 @@
+//
 import AxiosController from './axiosController';
 import { PagesInformation } from '../types/pagesInformation';
```

Comment:
A general optional comment regarding this module cohesion

I'd like to suggest taking a look at the page model conception (https://testcafe.io/documentation/402826/guides/concepts/page-model)

It helps reduce dependencies in code by moving page-related things into separate classes.
I see you use the raw HTML parsing approach -- maybe for the next projects, I propose to play around Testcase framework. It can run a browser in headless mode and interact with pages as a user would do

QA automation guys usually use this approach to test any kind of frontends (not only sites, even native desktop applications). Since here we have a classical scraping task, we can use it too